### ISSUE TITLE
#寻求帮助#如何提升ghostnet在place365数据集上的效果问题？

### ISSUE THREAD

**Issue by Geroge-W** [huawei-noah/ghostnet#45]
@iamhankai 您好，我尝试了将ghostnet网络在place365数据集上进行微调，在迭代了100个epoch之后，Acc@1: 35.% ,Acc@5: 62.%。
部分超参数设置如下：
batch_size = 256;
loss: CrossEntropyLoss();
optimizer: SGD();
lr_init=0.1 & Sets the learning rate to the initial LR decayed by 10 every 30 epochs

先只对最后的classifier训练，后面又对整个网络进行了训练，得到的两者的结果都很相近。
请问您能给一些在place365数据集上训练时的建议和帮助吗？
非常感谢！

**Comment by iamhankai** [comment]
我对place365数据集不太熟悉，有没有一些baseline的结果作为对比？

**Comment by Geroge-W** [comment]
> 我对place365数据集不太熟悉，有没有一些baseline的结果作为对比？

https://github.com/CSAILVision/places365
这儿有一些baseline的比较，非常感谢！

**Comment by iamhankai** [comment]
你自己复现过一些其他标准用语places365的网络吗？结果如何？看看是不是你的训练代码有问题？

**Comment by lld533** [comment]
places365测试用的是10-crop，而不是单个center-crop；Acc@1: 35.% ,Acc@5: 62.%可能是单个center-crop的结果（ImageNet的结果）。另外，finetune ImageNet上预训练的结果比从随机初始化开始训练的10-crop top-5结果要差2%左右（ResNet-50）。
